src/schema/objects/files.yaml

```
@@ -72,6 +72,11 @@ participants:
     followed by a list of optional columns describing participants.
     Each participant MUST be described by one and only one row.
 
+    If this file exists and a `phenotype/` folder is present, then
+    the `participant_id` entries MUST be the union of all subject directories
+    and all `participant_id` entries found among phenotypic and assessment data
+    in the `phenotype/` folder.
```

I would get rid of the conditional. If it doesn't exist, any rules don't apply.

I would also just say superset. $A \supseteq B \iff (A \cap B = B)$

```suggestion
    The `participant_id` entries MUST be a superset of all subject directories
    and all `participant_id` entries found among phenotypic and assessment data
    in the `phenotype/` folder.
```

If it's strongly desired, we can implement unions, but we can do this with a minimal change right now.